http/middleware/current_user.go

```diff
@@ -29,14 +29,14 @@ type UserStorer func(id uint) (User, error)
 // CurrentUser checks whether the "Accept" MIME type is "application/json"
```

Comment:
> CurrentUser checks whether the "Accept" MIME type is "application/json" and write a status code if so.

crusty?

Reply:
I think more just imprecise. `handleErr` below is where this toggling occurs.